### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by MohamedFBoussaid**

### CODE REVIEW

**Comment by MohamedFBoussaid** [comment]
Adding issue/feature templates

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```
Comment (M123-dev) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]: I think we are better of with only something like `package-version` and `platform`

**Review by M123-dev — APPROVED** [review]
Have a look at my comment, otherwise thanks for the PR @MohamedFBoussaid

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (MohamedFBoussaid) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Thanks @M123-dev , I have updated the description. Make it simpler and ask just for the package version.

Comment (M123-dev) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Thanks

**Pull request merged**